tasks/security_agent.py

```diff
@@ -102,3 +118,25 @@ def gen_mocks(ctx):
 
     with ctx.cd("./pkg/compliance"):
         ctx.run("./gen_mocks.sh")
+
+
+@task
+def functional_tests(
+    ctx, race=False, verbose=False, go_version=None, arch="x64", major_version='7', pattern='', output=''
+):
+    ldflags, gcflags, env = get_build_flags(ctx, arch=arch, major_version=major_version)
+
+    goenv = get_go_env(ctx, go_version)
+    env.update(goenv)
+
+    cmd = 'sudo -E go test -tags functionaltests {output_opt} {verbose_opt} {run_opt} {REPO_PATH}/pkg/security/tests'
```

Comment:
Is this running in CI?

Reply:
No, and it definitely should be ! This is a long running task that we have in the eBPF squad. The idea is to compile the functional tests as an executable and run them using Kitchen.